apps/testapp/src/components/RpcMethods/shortcut/sendShortcuts.ts

```diff
@@ -6,7 +6,7 @@ const ethSendTransactionShortcuts: ShortcutType[] = [
     key: 'Example Tx',
     data: {
       from: ADDR_TO_FILL,
-      to: ADDR_TO_FILL,
+      to: '0x0000000000000000000000000000000000000000',
```

Comment:
Could be good to have a comment along the lines of "Because users are blocked on smart wallets from having a `to` address of itself for security and safety reasons, using the universal blackhole address instead."